test-projects/node12/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "worker",
+    "version": "1.0.0",
+    "dependencies": {
+        "@adobe/asset-compute-sdk": "^1.0.2"
```

Comment:
Move to sdk v2

Reply:
I don't think this is required as part of the PR. None of the test workers in here use the errors.